packages/admin/dashboard/src/routes/orders/order-detail/components/order-summary-section/order-summary-section.tsx

```diff
@@ -716,21 +665,21 @@ const CostBreakdown = ({
 
           <div className="text-right">
             <Text size="small" leading="compact">
-              {getLocaleAmount(order.tax_total, order.currency_code)}
+              {getLocaleAmount(order.original_tax_total, order.currency_code)}
             </Text>
           </div>
         </div>
         {isTaxOpen && (
           <div className="flex flex-col gap-1 pl-5">
-            {Object.entries(taxCodes).map(([code, total]) => {
+            {taxes.map(({ type, total }) => {
               return (
                 <div
-                  key={code}
+                  key={type}
                   className="flex items-center justify-between gap-x-2"
                 >
                   <div>
                     <span className="txt-small text-ui-fg-subtle font-medium">
-                      {code}
+                      {t(`fields.${type}`)}
```

Comment:
nit: I noticed the tax and discount totals are bolded; is this intended?

<img width="1594" height="764" alt="CleanShot 2025-08-31 at 15 30 07@2x" src="https://github.com/user-attachments/assets/56f263bf-0255-4f12-a7dd-6f37924aaf88" />

Thinking if we should do one or the other.